app/helpers/projects_helper.rb

```diff
@@ -0,0 +1,29 @@
+module ProjectsHelper
+  def project_teams(project)
+    Team.where.not(id: project.teams.select(:id)) 
+  end
+
+  def isTeamAvailable?(project)
+    Team.where.not(id: project.teams.select(:id)).to_a.empty?
```

Comment:
Do not run sql queries in helpers or views. You can create a scope or method for this in model.
And use .exists? instead of to_a.empty?